examples/09-nat-gateways.yaml

```diff
@@ -0,0 +1,17 @@
+# An example of ClusterConfig object for customizing NAT gateway setup:
+--- 
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-2
+  region: eu-west-1
+
+vpc:
+  natGateway: single # valid options are disabled/single/highly-available
```

Comment:
Egress-only would be another option to consider, but no need to worry about it right now.

I was wondering if this should be something like
```
vcp:
   nat:
      gateway: <mode>
```
As we might need to cate for other fields, e.g. some users asked for a way to use pre-allocated EIPs...

Reply:
Egress-only gateway looks interesting too, but only supported with IPv6 so yeah, we can look at it later on.

```
 vcp:
   nat:
      gateway: <mode>
      ipList: <list of IPs>
```
Sure, something like ^^ makes sense and I guess I need to add some command line flags as well.